DigitalExcellence/dex-backend
# feature/368 seed update

## Description

Implementation for issue 368, updating the database record based on the seed

## Type of change


## Checklist

-   [  ] I did not update API Controllers, if I did, I added/changed Postman tests
-   [  ] I have performed a self-review of my own code

## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Do something here...
2. Then run this test...
3. After that, you should see the result here

https://github.com/DigitalExcellence/dex-backend/issues/368

Closes: #issuenumber
